src/storage/src/hummock/store/state_store.rs

```diff
@@ -278,6 +289,172 @@ impl HummockStorageCore {
         Ok(None)
     }
 
+    pub async fn iter_inner(
+        &self,
+        key_range: (Bound<Vec<u8>>, Bound<Vec<u8>>),
+        epoch: u64,
+        read_options: ReadOptions,
+    ) -> StorageResult<HummockStorageIterator> {
+        let compaction_group_id = self.get_compaction_group_id(read_options.table_id).await?;
+        // 1. build iterator from staging data
+        let (imms, sstable_infos, committed) = {
```

Comment:
suggest renameing `sstable_infos` to `uncommitted_ssts`.

Reply:
fixed